roles/variables/defaults/main.yml

```diff
@@ -1370,7 +1370,8 @@ control_center_additional_system_admins: "{{rbac_component_additional_system_adm
 
 # Secrets Protection Variables
 
-### Boolean to enable secrets protection on all components except Zookeeper
+### Boolean to enable secrets protection on all components except Zookeeper. Starting form CP 7.1.0 Secrets protection
+### works only with RBAC.
```

Comment:
why make this a hard requirement?

Reply:
Sorry, I think I answered in previous comment. 
RBAC is required. However, it doesn't have to be applied for all components. A simple admin user would serve the purpose to let CLI login to MDS.